src/FSharp.Control.TaskSeq/TaskSeq.fsi

```diff
@@ -725,6 +725,65 @@ type TaskSeq =
     /// <exception cref="T:ArgumentNullException">Thrown when the input task sequence is null.</exception>
     static member filterAsync: predicate: ('T -> #Task<bool>) -> source: TaskSeq<'T> -> TaskSeq<'T>
 
+    /// <summary>
+    /// Returns a task sequence that, when iterated, skips <paramref name="count" /> elements of the
+    /// underlying sequence, and then returns the remainder of the elements. Raises an exception if there are not enough
+    /// elements in the sequence. See <see cref="drop" /> for a version that does not raise an exception.
```

Comment:
```suggestion
    /// elements in the sequence. See <see cref="drop" /> for a version that does not raise an exception. <br />
```

Reply:
The `<br />` is not rendered (bug in F#). An empty line is, but adds a thin paragraph spacer.